Cleanup rpath of the shared libraies libhistory and libreadline.

This patch removes "/usr/lib" from rpath.

Signed-off-by: Samuel Martin <s.martin49@gmail.com>

---
diff -purN readline-6.2.orig/support/shobj-conf readline-6.2/support/shobj-conf
--- readline-6.2.orig/support/shobj-conf	2012-08-01 21:47:16.473274293 +0200
+++ readline-6.2/support/shobj-conf	2012-08-01 22:09:35.622686166 +0200
@@ -114,7 +114,7 @@ linux*-*|gnu*-*|k*bsd*-gnu-*|freebsd*-ge
 	SHOBJ_LD='${CC}'
 	SHOBJ_LDFLAGS='-shared -Wl,-soname,$@'
 
-	SHLIB_XLDFLAGS='-Wl,-rpath,$(libdir) -Wl,-soname,`basename $@ $(SHLIB_MINOR)`'
+	SHLIB_XLDFLAGS=`case $(libdir) in "/usr/lib") ;; *) echo '-Wl,-rpath,$(libdir) -Wl,-soname,\`basename $@ $(SHLIB_MINOR)\`'`
 	SHLIB_LIBVERSION='$(SHLIB_LIBSUFF).$(SHLIB_MAJOR)$(SHLIB_MINOR)'
 	;;
 
